0.25.4 release notes (#407)
astronomer/docs: cloud/stable/07_resources/01_release-notes.md

## Astronomer v0.25

### v0.25.4

> Release Date: June 31st, 2021

Reviewer: Should this be today (July 2nd, 2021)?

> Release Date: June 31st, 2021

Reviewer: ```suggestion
Release Date: July 2nd, 2021
```

#### Bug fixes

> - Fixed an issue where Workspace Admin service accounts could not log in to Astronomer.

Reviewer 2: could not CRUD (didn't have access)

> - Fixed an issue where Workspace Admin service accounts could not log in to Astronomer.

Reviewer: ```suggestion
- Fixed an issue where Workspace Service Accounts lost access to Astronomer.
```

(Don't think it was a problem exclusive to Admin - I think it was all Workspace SAs)
